packages/Webkul/Customer/src/Rules/VatIdRule.php

```diff
@@ -37,6 +37,6 @@ public function passes($attribute, $value)
      */
     public function message()
     {
-        return trans('shop::app.invalid_vat_format');
+        return trans('admin::app.customers.addresses.invalid-format');
```

Comment:
Please use the language from the customer package in accordance with our standard.

Reply:
Done.